study-builder/studies/testboston/i18n/es.conf

```diff
@@ -25,23 +25,23 @@
     "s2_purpose_title": "¿Por qué se realiza este estudio de investigación?",
     "s2_purpose_p1": "En este estudio nos proponemos averiguar más sobre la cantidad de adultos de Massachusetts que tienen COVID-19. Queremos saber qué tan habitual es que la población se contagie de COVID-19 y no presente síntomas. Trataremos de averiguar más sobre cómo calcular mejor la tasa de infección. Estudiaremos cómo hacer un seguimiento de la tasa de infección y qué hacer si hay nuevos brotes de COVID-19 en los próximos meses. También queremos saber si las personas que se contagian de COVID-19 tienen menos probabilidades de contagiarse por segunda vez. En este estudio también se desarrollará un sistema aceptable para implementar pruebas de detección de COVID-19 en el hogar, de modo que sea posible analizar a una gran cantidad de personas.",
     "s2_timing_title": "¿Cuánto tiempo participará en este estudio de investigación?",
-    "s2_timing_p1": "[ES] If you decide to join this research study, it will take you about <strong>6 months</strong> to complete the study. During your study participation, we will not ask you to make any study visits to Brigham and Women's Hospital.",
+    "s2_timing_p1": "Si decide incorporarse a este estudio de investigación, le llevará <strong>6 meses</strong> realizarlo en su totalidad. [ES: During your study participation, we will not ask you to make any study visits to Brigham and Women's Hospital.]",
```

Comment:
New convention: for longer text where we're slicing/dicing and only replacing a sentence or two, we're wrapping that part with `[LANG: ... ]`.